include/x86_64/gdt/gdt.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
```

Comment:
namespacing?

Reply:
wdym

Reply:
this entire file isn't in a namespace

Reply:
I didn't mean for them to be accessed via core::gdt but I can change it